config/routes.rb

```diff
@@ -3,6 +3,7 @@
     get 'users/sign_out', to: "devise/sessions#destroy", as: "logout"
   end
   devise_for :users
+  # devise_for :users, skip: :all
```

Comment:
commented code is something that is better be avoided. It stays forever and everyone is afraid to remove it. Id you don't need this, remove. After all there is git keeping all the history.